airbyte-integrations/connectors/source-tiktok-marketing/integration_tests/abnormal_state.json

```diff
@@ -10,42 +10,50 @@
   },
 
   "ads_reports_daily": {
+    "stat_time_day": "2030-01-01",
```

Comment:
I believe we should handle this migration gracefully in the code in order not to brake incremental syncs

Reply:
Updated PR